### ISSUE TITLE
tap-marketo not working with target-csv

### ISSUE THREAD

**Issue by nicholassewitz** [singer-io/target-csv#5]
When I try and send marketo data to the singer target-csv I am getting this error 

```
artsy-nicholass:marketo-fulcrum nicholassewitz$ tap-marketo -c config.json -s state.json | target-csv -c csv-config.json
Traceback (most recent call last):
  File "/usr/local/bin/tap-marketo", line 7, in <module>
    from tap_marketo import main
  File "/usr/local/lib/python3.6/site-packages/tap_marketo/__init__.py", line 12, in <module>
    from singer import utils
ImportError: cannot import name 'utils'
```

**Comment by nicholassewitz** [comment]
fyi when I ran on separate virtualenv's this error went away.

**Comment by nicholassewitz** [comment]
now getting this error

```
    sys.exit(main())
  File "/Users/nicholassewitz/marketo-fulcrum/tap-marketo/lib/python3.6/site-packages/tap_marketo/__init__.py", line 320, in main
    do_sync()
  File "/Users/nicholassewitz/marketo-fulcrum/tap-marketo/lib/python3.6/site-packages/tap_marketo/__init__.py", line 267, in do_sync
    singer.write_schema("leads", schema, ["id"])
  File "/Users/nicholassewitz/marketo-fulcrum/tap-marketo/lib/python3.6/site-packages/singer/__init__.py", line 91, in write_schema
    key_properties=key_properties))
  File "/Users/nicholassewitz/marketo-fulcrum/tap-marketo/lib/python3.6/site-packages/singer/__init__.py", line 52, in _write_message
    sys.stdout.write(message.tojson() + '\n')
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by joaomarcosgris** [comment]
Have you tried to install the latest version of singer-python? When I installed target-csv it broke the tap-adwords and that was the issue. I've made a PR #6 that fix this by changing the setup.py to require the latest version.

**Comment by nicholassewitz** [comment]
I ended up just interacting directly with the api.

**Comment by joaomarcosgris** [comment]
I don't have a marketo account, but I thin this was fixed in version 0.2.3. Can you check that @cmerrick?

**Comment by cmerrick** [comment]
this is more of a design issue that's best fixed by using different virtualenvs.  That's an issue because it's not very user friendly.  Closing for now, but that more general issue is on our radar.
